src/app/components/Timestamp/index.jsx

```diff
@@ -11,6 +11,13 @@ import {
 const formatTimestamp = timestamp => {
   const dateTime = new Date(timestamp * 1000);
 
+  // if the date is invalid return null - https://stackoverflow.com/questions/1353684/detecting-an-invalid-date-date-instance-in-javascript#answer-1353711
+  if (
+    !(dateTime instanceof Date && !isNaN(dateTime)) // eslint-disable-line no-restricted-globals
```

Comment:
Should be okay to simplify to `if (isNaN(dateTime))`, since that captures all the examples - if dateTime is Invalid Date, and if it the original timestamp is null or undefined.